src/views/Onboarding/Sections/Playgrounds/Playgrounds.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+import classNames from "classnames";
+import withStyles from "@material-ui/core/styles/withStyles";
+import Close from "@material-ui/icons/Close";
+
+import componentsStyle from "../../../../assets/jss/material-kit-react/views/components";
+import PlaygroundSearch from "./PlaygroundSearch";
+import PlaygroundMap from "./PlaygroundMap";
+import GridContainer from "../../../../components/Grid/GridContainer";
+import GridItem from "../../../../components/Grid/GridItem";
+import PlaygroundCta from "./PlaygroundCta";
+import WorkspaceWelcomeContent from "../../../Workspace/Sections/WorkspaceWelcome/WorkspaceWelcomeContent";
+
+class Playgrounds extends Component {
+    render() {
+        const {
+            playgrounds,
+            classes,
+            onCtaClick,
+            playground,
+            map,
+            user,
+            handlePlaygroundChange,
+        } = this.props;
+
+        const defaultPlayground = {
+            default: true,
+        };
+
+        return (
+          <div className={classNames(classes.main) + " onboarding-container"}>
+              <div className={classes.playgroundsTitle}>Zoek een speeltuin in jouw buurt een doe mee!</div>
+              <div className={classes.playgroundsBg} />
+              <GridContainer className={"grid-container-map"}>
```

Comment:
I think we should enable ctrl+zoom again as there is no room to scroll anymore; also because of the CTA's we dont want the user to search on this map anymore

Reply:
Should ctrl+zoom be enabled only for this map of for AddPlayground's map too?

Reply:
I enabled it.